airbyte-webapp/src/pages/SettingsPage/pages/ConnectorsPage/components/CreateConnectorModal.tsx

```diff
@@ -117,7 +122,7 @@ const CreateConnectorModal: React.FC<IProps> = ({ onClose, onSubmit, errorMessag
           }}
           validateOnBlur
           validateOnChange
-          validationSchema={validationSchema}
+          validationSchema={isCloudApp() === false ? standardValidationSchema : customConnectorValidationSchema}
```

Comment:
In general, I would use the `FeatureService` over `isCloudApp()`, but the `AllowCustomImageUpload` doesn't tell us if a user is on Cloud specifically.  Open to a better check if someone has one!